runtimes.yaml

```diff
@@ -25,8 +25,8 @@
       version: ">=22"
       brew: node@22
       url: https://nodejs.org/en/download
-- name: "Python with uv"
-  description: "Python with the extremely fast Python package and project manager, uv."
+- name: "Python + uv"
```

Comment:
The "with uv" casing just looks weird, makes it look like we forgot to uppercase "uv". IMO we should go with "+"... more closely matches our brand language, e.g. the "CrewAI + Agentuity" we do on the website. Keep it ASCII-ish characters when we can.